tests/types/vcpkg-test.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "https://packageurl.org/schemas/purl-test.schema-1.0.json",
+  "tests": [
+    {
+      "description": "simple vcpkg purl",
+      "test_group": "base",
+      "test_type": "parse",
+      "input": "pkg:vcpkg/ms-gsl@4.0.0",
+      "expected_output": {
+        "type": "vcpkg",
+        "namespace": null,
+        "name": "ms-gsl",
+        "version": "4.0.0",
+        "qualifiers": null,
+        "subpath": null
+      },
+      "expected_failure": false,
+      "expected_failure_reason": null
+    },
+    {
+      "description": "vcpkg with port_revision",
+      "test_group": "base",
+      "test_type": "parse",
+      "input": "pkg:vcpkg/ms-gsl@4.0.0?port_revision=4",
```

Comment:
Where would the port_revision come from? I could not find anything in https://github.com/microsoft/vcpkg/tree/master/ports/ms-gsl

Reply:
The default is 0, see https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#port-version. It can be found in the port's `vcpkg.json`, see e.g. https://github.com/microsoft/vcpkg/blob/master/ports/bzip2/vcpkg.json#L4